Description: Upstream tarball vs git tag
 Upstream tarball doesn't match with the git tag
 .
 libdigest-sha1-perl (2.13-1~gekkio1) trusty; urgency=low
 .
   * Initial Release.
Author: Joonas Javanainen <joonas.javanainen@gmail.com>

--- libdigest-sha1-perl-2.13.orig/MANIFEST
+++ libdigest-sha1-perl-2.13/MANIFEST
@@ -11,4 +11,3 @@ t/badfile.t
 t/bits.t
 t/sha1.t
 typemap
-META.yml                                 Module meta-data (added by MakeMaker)
--- /dev/null
+++ libdigest-sha1-perl-2.13/MANIFEST.SKIP
@@ -0,0 +1,16 @@
+^blib/
+\.old$
+\.bak$
+Makefile$
+^MANIFEST.SKIP$
+^#
+/#
+~$
+\b(RCS|CVS)/[^/]+$
+\.cvsignore$
+pm_to_blib$
+\.o$
+\.c$
+\.bs$
+^Digest-SHA1-.*\.tar\.gz$
+^other/
